roles/manage_dbserver/tasks/generate_password.yml

```diff
@@ -5,26 +5,34 @@
     mode: 0700
     state: directory
   become: no
+  run_once: true
+  no_log: "{{ disable_logging }}"
   delegate_to: localhost
 
 - name: Create passfile location
   set_fact:
-    passfile: "{{ pass_dir }}/{{ inventory_hostname }}_{{ input_user }}_pass"
```

Comment:
When I introduced the node name into the pass file name, my goal was to ensure to have different passwords (for the same user name) when deploying multiple primary nodes or pem servers.

Do you think this is fine if we use the same random password for every nodes ?

Reply:
I did that intentionally because I noticed for each standby replication user was having a different password. With this, I had included `run_once: true` in manage_dbserver/tasks/generate_password.yml`

Reply:
also for replication, we are using one standard user `repuser`

Reply:
I see, thank you. I'll apply some changes on #77